src/components/templates/PosterHall/components/PosterPreview/PosterPreview.scss

```diff
@@ -0,0 +1,74 @@
+@import "scss/constants";
+
+.poster-preview {
+  display: flex;
+  flex-direction: column;
+
+  height: 400px;
+  width: 300px;
+
+  padding: 15px;
+  margin: 10px;
+  border-radius: 10px;
+
+  background-color: $saturated-black-alt;
+  color: $white;
+
+  cursor: pointer;
+
+  &:hover {
+    opacity: 0.8;
+  }
+
+  &--live {
+    background-color: $primary;
+  }
+
+  &__pdf {
+    border-radius: 10px;
+    overflow: hidden;
+    position: relative;
+    background-color: lightgray;
+    height: 50%;
+    overflow: hidden;
+  }
+
+  &__pdf-iframe {
+    height: 100%;
+    width: 100%;
+    border: none;
+  }
+
+  &__title {
+    font-size: 14px;
+    font-weight: 600;
+
+    margin: 10px 0;
+  }
+
+  &__categories {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  &__category {
+    display: flex;
+    font-size: 10px;
+    color: $white;
+    cursor: pointer;
+    padding: 7px;
+    margin: 3px;
+    border-radius: 15px;
+  }
+
+  &__author {
+    margin-top: auto;
+    &__name {
+      font-size: 12px;
+      font-weight: 500;
+    }
+    &__institution {
+      font-size: 10px;
+    }
+  }
```

Comment:
Moved them into root scope in https://github.com/sparkletown/sparkle/pull/1299/commits/f4c868471dcad8c0b597c686ef3a796ef5c4aa89

I don't think they can be reused anywhere now, but we probably will get to a level, when we have defined title/subtitle/paragraph classes with certain styles.

Reply:
I wasn't meaning 'generally across the entire platform', but I was meaning generally within the current component/styles. eg. 2 'title like elements' that currently have custom BEM names, that could both be solved by the same class.

Reply:
It only has an author name now. Guess resolved?

Reply:
Yup. I believe it was fine since https://github.com/sparkletown/sparkle/pull/1299#discussion_r623358398, I was just explaining what I meant in https://github.com/sparkletown/sparkle/pull/1299#discussion_r623562734 because you didn't seem to understand me.